### ISSUE TITLE
Cannot read property 'isOperator' of undefined

### ISSUE THREAD

**Issue by DeoLukamba** [VSCodeVim/Vim#6341]
**To Reproduce**

1.  Go to '...'
2.  Click on '....'
3.  Scroll down to '....'
4.  See error

**VSCodeVim version**: 1.19.0

<details><summary>Stack trace</summary>

```
TypeError: Cannot read property 'isOperator' of undefined
    at c:\Users\Deogratias Lukamba\.vscode\extensions\vscodevim.vim-1.19.0\out\extension.js:2:779767
    at Array.filter (<anonymous>:null:null)
    at a.get operators [as operators] (c:\Users\Deogratias Lukamba\.vscode\extensions\vscodevim.vim-1.19.0\out\extension.js:2:779754)
    at a.get operator [as operator] (c:\Users\Deogratias Lukamba\.vscode\extensions\vscodevim.vim-1.19.0\out\extension.js:2:779675)
    at a.isOperatorPending (c:\Users\Deogratias Lukamba\.vscode\extensions\vscodevim.vim-1.19.0\out\extension.js:2:780484)
    at m.get currentModeIncludingPseudoModes [as currentModeIncludingPseudoModes] (c:\Users\Deogratias Lukamba\.vscode\extensions\vscodevim.vim-1.19.0\out\extension.js:2:788363)
    at I.handleKeyEvent (c:\Users\Deogratias Lukamba\.vscode\extensions\vscodevim.vim-1.19.0\out\extension.js:2:740641)
    at Object.promise (c:\Users\Deogratias Lukamba\.vscode\extensions\vscodevim.vim-1.19.0\out\extension.js:2:383783)
    at runMicrotasks (<anonymous>:null:null)
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
    at Object.runTasks (c:\Users\Deogratias Lukamba\.vscode\extensions\vscodevim.vim-1.19.0\out\extension.js:2:794795)

```

</details>

**Comment by J-Fields** [comment]
Fixed in 1.19.1, please upgrade

**Comment by DeoLukamba** [comment]
Thanks a lot @J-Fields . It worked
